### DISCUSSION TITLE
New module nearly ready :)

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1937] [Show and tell]
Full form designer, data capture and presenter.

![Animation](https://user-images.githubusercontent.com/9447612/149517443-71e81b24-23c7-4f54-a640-1632fe891e56.gif)

I'm very excited by this module and will be making it available on request for testing.

**Comment by lanthonyneville** [comment]
This looks amazing! Thanks Leigh .... this is just the kind of thing to give Oqtane a boost.
What is the underlying data storage?

**Reply by leigh-pointer** [reply]
@lanthonyneville Thanks ^5 
There is still a bit to do as far a the data presentation layer goes, but I'm working on that now.  I have a couple more tools for the toolbox to build but hope to go to a beta version very soon.

> What is the underlying data storage?
The storage is json as this best suits the job.  So far no problems but I will need to get this stress tested,

**Reply by ADefWebserver** [reply]
This is the potential of Oqtane being realized 👍🏾🙌🏾

**Comment by leigh-pointer** [comment]
![FullDesigner](https://user-images.githubusercontent.com/9447612/149632829-231bd7ca-38f2-4cf3-8a60-0e5b4574f3ec.png)

**Comment by leigh-pointer** [comment]
2300+ records added and presented
![AnimationData](https://user-images.githubusercontent.com/9447612/149807542-c2f33e47-fba0-4ae6-9756-21ecaa5f679b.gif)

**Comment by leigh-pointer** [comment]
I have just been playing with the Presenter part of the module and getting some pretty cool results.

![AnimationPresenter](https://user-images.githubusercontent.com/9447612/151230652-724a1734-b11c-4a19-98af-fa0ac0101b00.gif)
